crates/bevy_scene/src/command.rs

```diff
@@ -19,7 +19,12 @@ impl Command for SpawnScene {
     }
 }
 
+// Written in "one line" to insert it into the module overview.
+#[doc = "[`Commands`] extension: <ul>\
+<li><code>.[spawn_scene](SpawnSceneCommands::spawn_scene)(scene: [Handle]<[Scene]>)</code></li>\
+</ul>"]
```

Comment:
This definitely looks a bit messy in code (and unfortunately means rust-analyzer doesn't provide specific highlighting inside this doc attribute), but I checked that the warnings for broken links work as expected, at least.

The code is a tradeoff to enable these easily-skimmable item summaries in the module documentation:
![image](https://user-images.githubusercontent.com/715360/122056626-0da6ab00-cdea-11eb-85a3-c48b091db266.png)

I think it's useful to have this information visible at the module level since it's quite short but provides a lot of information about the data flow when using this module.